Tests do not run in a remote machine due to matplotlib

For example

```
emirdrp/tests/test_drp.py:17: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
//work/gcsop/lib/python/eggs/lib64/numina-0.14.dev0-py2.7-linux-x86_64.egg/numina/core/objimport.py:31: in import_object
    mm = importlib.import_module(mods)
/usr/lib64/python2.7/importlib/__init__.py:37: in import_module
    __import__(name)
emirdrp/recipes/aiv/__init__.py:35: in <module>
    from emirdrp.products import MasterBias
emirdrp/products.py:30: in <module>
    from numina.array.wavecal.slitlet import Slitlet
//work/gcsop/lib/python/eggs/lib64/numina-0.14.dev0-py2.7-linux-x86_64.egg/numina/array/wavecal/slitlet.py:8: in <module>
    import matplotlib.pyplot as plt
/usr/lib64/python2.7/site-packages/matplotlib/pyplot.py:97: in <module>
    _backend_mod, new_figure_manager, draw_if_interactive, _show = pylab_setup()
/usr/lib64/python2.7/site-packages/matplotlib/backends/__init__.py:25: in pylab_setup
    globals(),locals(),[backend_name])
/usr/lib64/python2.7/site-packages/matplotlib/backends/backend_gtkagg.py:10: in <module>
    from matplotlib.backends.backend_gtk import gtk, FigureManagerGTK, FigureCanvasGTK,\
/usr/lib64/python2.7/site-packages/matplotlib/backends/backend_gtk.py:13: in <module>
    import gtk; gdk = gtk.gdk
/usr/lib64/python2.7/site-packages/gtk-2.0/gtk/__init__.py:64: in <module>
    _init()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def _init():
        import sys
    
        sys_path = sys.path[:]
    
>       _gtk.init_check()
E       RuntimeError: could not open display

```

--- a/emirdrp/recipes/image/shared.py
+++ b/emirdrp/recipes/image/shared.py
@@ -34,7 +34,6 @@ from astropy.visualization.mpl_normalize import ImageNormalize
 from scipy.spatial import KDTree as KDTree
 
 import matplotlib as mpl
-import matplotlib.pyplot as plt
 from matplotlib.patches import Ellipse
 from matplotlib.collections import PatchCollection
 
@@ -157,6 +156,8 @@ class DirectImageCommon(EmirRecipe):
 
     def __init__(self, *args, **kwds):
         super(DirectImageCommon, self).__init__(version=__version__)
+        # Required to delay the backend initialization (issue numina/#102, #49)
+        import matplotlib.pyplot as plt
 
         self._figure = plt.figure(facecolor='white')
         self._figure.canvas.set_window_title('Recipe Plots')

